pkg/reconciler/certificate/certificate.go

```diff
@@ -207,7 +208,7 @@ func (c *Reconciler) reconcileCMCertificate(ctx context.Context, knCert *v1alpha
 
 func (c *Reconciler) updateStatus(existing *v1alpha1.Certificate, desired *v1alpha1.Certificate) error {
 	existing = existing.DeepCopy()
-	return reconciler.RetryUpdateConflicts(func(attempts int) (err error) {
+	return pkgreconciler.RetryUpdateConflicts(func(attempts int) (err error) {
```

Comment:
Can we remove the original `RetryUpdateConflicts` now?

Reply:
sorry, thought i had, done. Thanks